pkg/handler/processor/guesser/type_ite6.go

```diff
@@ -39,7 +39,7 @@ func (_ *ite6TypeGuesser) GuessDocumentType(blob []byte, format processor.Format
 				return processor.DocumentITE6Generic
 			} else if strings.HasPrefix(statement.PredicateType, "https://in-toto.io/attestation/certify/v0.1") {
 				return processor.DocumentITE6Generic
-			} else if strings.HasPrefix(statement.PredicateType, "https://in-toto.io/attestation/vuln/v0.1") {
+			} else if strings.HasPrefix(statement.PredicateType, "https://in-toto.io/attestation/vulns/v0.1") {
```

Comment:
it will also have to be changed here: https://github.com/guacsec/guac/blob/main/pkg/certifier/attestation/attestation_vuln.go#L30

Until we use the upstream attestation....

Reply:
it looks like [intoto gol](https://github.com/in-toto/in-toto-golang/tree/master/in_toto) needs to be updated to include the predicate as well?

Reply:
oh it looks like there's the other repo https://github.com/in-toto/attestation/tree/main/go/predicates..

Reply:
can i follow up with these changes in a separate PR? i'd like to get at least the flow to the parser first.. thoughts?